### PR TITLE
clean up preprocessor defines

### PR DESCRIPTION
**Pull request by Qix-**
Should be a little easier to look at here.

Try to keep headers at the top. Most of the time it doesn't functionally matter because people write their header files correctly. However, you shouldn't rely on that. Plus it's a readability thing; if I don't see a header at the top of a translation unit then I assume it isn't needed by the unit at all.

Don't be afraid to use more preprocessor macros if they're justified :v:

### CODE REVIEW

File: binding.cc [hunk 1]
```diff
@@ -9,31 +9,30 @@ using namespace v8;
...
+#	include <windows.h>
+#	include <security.h>
+#	include <secext.h>
+#	include <lmcons.h>
+#	include <tchar.h>
+#else
+#	include <sys/types.h>
+#	include <unistd.h>
+#	include <pwd.h>
+#endif
```
Comment (sindresorhus) [inline binding.cc]: Can you place this ifdef above `#include <node.h>`?

Comment (Qix-) [inline binding.cc]: Yeah I didn't even see that, good catch.

**Comment by sindresorhus** [comment]
Thank you :D

**Comment by sindresorhus** [comment]
:wine_glass:

**Comment by Qix-** [comment]
So much :wine_glass:

**Pull request merged**